modules/cms/lang/en/lang.php

```diff
@@ -77,7 +78,7 @@
         'export_folders_label' => 'Folders',
         'export_folders_comment' => 'Please select the theme folders you would like to export',
         'delete_button' => 'Delete',
-        'delete_confirm' => 'Are you sure you want to delete this theme? It cannot be undone!',
```

Comment:
Because of: https://github.com/octobercms/october/commit/d6a7fdc74c8726149358545aff4e360a01bdf4fb#diff-73ae1299d2c352fefb63eb86c178f48dL114